filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,21 @@
+# xfstests test suite
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+## How to run it
+
+### Configure which test cases to run, e.g.
+```bash
+$ export TEST_PARAM_FSTYPE=xfs
+$ export TEST_PARAM_RUNTESTS=generic/001
```

Comment:
Could you put this section right before the "Execute the test" section? I think it relates to it closer than to the "Dependencies".